includes/classes/Constants.php

```diff
@@ -0,0 +1,15 @@
+<?php class Constants {
+
+    public static $passwordsDoNotMatch = "Both password need to match";
+    public static $passwordTooSimple = "Password needs to consist of a lowercase, uppercase, numeric and a \"! or ?\"";
+    public static $emailInvalid = "Email is invalid";
+    public static $emailDoNotMatch = "Both the email need to match";
+    public static $nameCharacterLength = "Your First and Last Name must be between 2 and 25 characters";
+    public static $usernameCharacterLength = "Your username must be between 5 and 25 characters";
```

Comment:
stored as static, so that it can be accessed even if without instantiating the class